src/components/TokensCard/TokensCard.tsx

```diff
@@ -37,16 +39,23 @@ export const TokensCard: FC<TTokensCard> = ({ collectionId, tokenId, testid, ...
     auction,
     video
   } = useMemo<Partial<Offer & NFTToken & NFTCollection>>(() => {
```

Comment:
still merging

you can calculate name for token in actual components instead (should be only two places). Usign types merging (especially when u mix it with offer) is a dangerous path.